### ISSUE TITLE
Add the option to run a 'Process Default' on Save

### ISSUE THREAD

**Issue by samusaran** [TabularEditor/TabularEditor3#102]
As title, it could be useful on some models to have the Process Default run every time the changed are saved to a workspace.

I would put it off by default, and maybe have it set on a per-model basis.

**Comment by otykier** [comment]
I agree this should be a per-model setting, as any processing operation could potentially take a long time to complete (and you will not be able to save the model subsequently, while a processing operation is under way). Will definitely consider for a future release.

**Comment by samusaran** [comment]
I had a bit more time to think about my use case, and one additional option could be to run "Process Recalc" on the model on save.

The original reason I made this issue was because every time I add a new relationship or calculated column\table with tabular editor 3 and save to the server, the object always needs recalculation, leading to errors if someone tries to read data from the model (I usually work on tabular models, while other people develop pbix while connected to the dev server).

So instead of Process Default, Process Recalc should be enough for this use case. I would still leave both choices up to the user.

**Comment by marcosqlbi** [comment]
I agree with @samusaran - choosing between Process Default and Process Recalc as automatic action saving to the workspace requires a very good understanding of the differences between the two options, but it could be a welcome feature. It just need a good documentation!
